test/import-index.js

```diff
@@ -16,9 +16,17 @@ const error = {
 	message: 'Do not reference the index file directly.'
 };
 
+const ignoreImportsOptions = [{
+	ignoreImports: true
+}];
+
 ruleTester.run('import-index', rule, {
 	valid: [
 		'const m = require(\'.\')',
+		{
+			code: 'const m = require(\'.\')',
```

Comment:
you might mean to  test `require(./index)`

Reply:
done